app/src/hooks/eventHooks.ts

```diff
@@ -25,15 +25,19 @@ export const useEvent = (id: string | undefined): [IEvent | undefined] => {
   return [event];
 };
 
-export const useRecentlyCreatedEvent = (): {
-  createdEventId: string | undefined;
+export const useCreatedEvents = (): {
+  createdEventIds: string[];
   setCreatedEventId: (string: string) => void;
 } => {
   const [storage, setStorage] = useLocalStorage({
-    key: 'recently-created-event',
+    key: 'created-events',
   });
+  const parsedStorage: string[] = storage ? JSON.parse(storage) : [];
+  const updateStorage = (string: string) =>
+    JSON.stringify([...parsedStorage, string]);
+
   return {
-    createdEventId: storage,
-    setCreatedEventId: setStorage,
+    createdEventIds: parsedStorage,
+    setCreatedEventId: (string: string) => setStorage(updateStorage(string)),
```

Comment:
Denne funksjonaliteten startet jo som en liten 'nice to have'-greie, så arrangører skulle slippe å gå via mailen for å redigere. Nå har jo scopet økt litt, så jeg tenkte bare på en liten greie:
Om en bruker fjerner hele local storage, vil da brukeren aldri kunne redigere eventene uten å gå inn via linken i mailen?

Vil vi, på sikt, synce dette med databasen? Så 'dine opprettede arrangementer' lista kan oppdateres?

Reply:
Godt poeng. 
Jeg og Tarjei snakka litt om dette, og da må man i så fall lagre e-post i tokenet så man kan gi tilgang til folk. Absolutt mulig, men tror for MVPen at det holder med unik lenke - også fordi flere folk enkelt kan få tilgang om det trengs. Men absolutt noe å tenke på til senere.